File: Tellem/src/main/java/com/example/tellem/application/models/Project.java
package com.example.tellem.application.models;

import java.util.ArrayList;
import java.util.List;

public class Project {
    private String name;
    private List<TaskList> lists = new ArrayList<>();

    public Project(String nameTmp){
        this.name = nameTmp;
    }

    public void setName(String nameTmp){
        name = nameTmp;
    }
    public String getName(){
        return name;
    }

    public void addTaskList(TaskList taskList){
        this.lists.add(taskList);
    }

    public void removeTaskList(int id){
        this.lists.remove(id);
    }

    public int taskListSize(){
        return lists.size();
    }

    public List<TaskList> getTaskLists(){
        return lists;
    }
}

File: Tellem/src/main/kotlin/tellem/model/Project.kt
package tellem.model

import tellem.model.TaskList

class Project(var name: String, var description: String, var pID: Int?) {
    var taskLists: MutableList<TaskList> = ArrayList()

    fun addTaskList(taskList: TaskList) {
        this.taskLists.add(taskList)
    }

    fun removeTaskList(id: Int) {
        this.taskLists.removeAt(id)
    }

    fun taskListSize(): Int {
        return taskLists.size
    }
}